cliquet/resource.py

```diff
@@ -198,7 +192,14 @@ def register_service(endpoint_type):
         name = viewset.get_service_name(endpoint_type, resource)
 
         service = Service(name, path, depth=depth,
-                          **viewset.service_arguments or {})
+                          **viewset.get_service_arguments())
+
+        # Attach viewset and resource to the service for later reference.
+        service.viewset = viewset
+        service.resource = resource
```

Comment:
Here, it's not possible to add [a reified property](http://docs.pylonsproject.org/projects/pyramid/en/latest/api/decorator.html.) to the request just because … we don't have access to it just here! Do you have something else in mind?